dali/pipeline/operators/reader/nvdecoder/nvdecoder.cc

```diff
@@ -36,6 +36,9 @@
 
 namespace dali {
 
+static constexpr int kNvcuvid_success = 1;
```

Comment:
Maybe enum?

Reply:
Since we only have two possible returns, isnt it better to stay with static constexpr?